bin/jira.py

```diff
@@ -0,0 +1,329 @@
+#!/usr/bin/env python3
+"""
+A minimal CLI for interacting with our day to day Jira tasks.
+
+This script expects a ~/.jira_credentials file to exist with the following
+format:
+```yaml
+user: "<your canonical email address>"
+token: "<a valid Jira API token>"
```

Comment:
It errors out when the optional arguments are missing now

```
$ ./jira.py list-epics --backlog=UDENG
Traceback (most recent call last):
  File "/tmp/desktop-engineering/bin/./jira.py", line 319, in <module>
    client = JiraClient(Credentials(**yaml.safe_load(f)))
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Credentials.__init__() missing 2 required positional arguments: 'board_id' and 'sprint_prefix'
```

seems you made the board_id and sprint_prefix mandatory for the python script now, so either they need to be made optional again or the expected yaml description needs to be synced from the other script